src/Makefile.am

```diff
@@ -142,7 +142,7 @@ souffle_sources = \
 noinst_LTLIBRARIES = libsouffle.la
 libsouffle_la_SOURCES  = $(souffle_sources)
 libsouffle_la_CXXFLAGS = $(souffle_CPPFLAGS)
-libsouffle_la_LDFLAGS = --static --dlopen --pic
+libsouffle_la_LDFLAGS = --static --dlopen --pic -ldl -lffi
```

Comment:
The libraries should be added to LIBS by the configure script, if AC_CHECK_LIB was run

Reply:
They are requirements for the interpreter. They cannot be made optional.